### PR TITLE
By default don't scroll text if it fits on the displays

### LINKED ISSUE #29: Showing text without scrolling
Is there currently a way to show some string centered (without scrolling) if it fits on the displays? It might be useful for showing things like the current time or temperature where you don't want the text to scroll.

If not, I could send you a Pull Request if you like.

### PR DESCRIPTION
**Pull request by bauerj**
This changes the behaviour of `matrix.show_message` to do the following:

1. Check if the rendered text is longer than all the displays combined
2. If it **is** longer:
    1.  Proceed as usual
3. If it's not longer:
    1. Calculate the margin needed to center the text
    2. Display the text with the margin as offset and flush immediately.

It looks like this: img_20160903_214349

Closes #29

### CODE REVIEW

File: max7219/led.py [hunk 1]
```diff
@@ -415,23 +415,34 @@ def scroll_down(self, redraw=True):
...
+        src = [c for ascii_code in text for c in font[ord(ascii_code)]]
+        scroll = always_scroll or len(src) > display_length
+        if scroll:
+            # Add some spaces on (same number as cascaded devices) so that the
+            # message scrolls off to the left completely.
+            src += [c for ascii_code in ' ' * self._cascaded
+                    for c in font[ord(ascii_code)]]
+        else:
+            # How much margin we need on the left so it's centered
+            margin = int((32 - len(src))/2)
```
Comment (rm-hull) [inline max7219/led.py]: Should the `32` be `display_length` ?

**Comment by rm-hull** [comment]
Other than above comment, looks good 👍

File: max7219/led.py (hunk 1 above)
Comment (bauerj) [inline max7219/led.py]: Whoops, good catch!

**Comment by bauerj** [comment]
Okay, I changed that :tada:

**Pull request merged**